src/mono/browser/test-main.js

```diff
@@ -312,6 +321,12 @@ async function run() {
         App.runtime = await dotnet.create();
         App.runArgs = runArgs
 
+        if (ENVIRONMENT_IS_WEB && isFirefox) {
+            console.log("Application arguments: " + runArgs.applicationArguments.join(' '));
```

Comment:
It's a duplication of code from line 308 (for Firefox).
```suggestion
```

Reply:
In firefox we would not see any `console.log` until the console proxy was installed.